lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,11 +1,11 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
+  <div class="sidebar-body">
```

Comment:
Please explain how the changes to the class here are relevant to the top month being empty.